nlp-common/src/main/java/io/cdap/google/common/AnalyzeEntities.java

```diff
@@ -30,6 +33,34 @@ public AnalyzeEntities(String languageCode, EncodingType encoding, LanguageServi
     super(languageCode, encoding, language);
   }
 
+  @Override
+  protected StructuredRecord getRecordFromJson(String json) {
+    Schema schema = Schema.recordOf(AnalyzeEntities.class.getName(),
```

Comment:
these schemas should be constants, otherwise each individual record will contain its own schema object, which will use much more memory than needed.